lib/presentation/widgets/man_widget.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:flutter/material.dart';
+
+class ManWidget extends StatelessWidget {
+  const ManWidget({Key? key}) : super(key: key);
+
+  static const manImagePath = "assets/images/Man1.png";
+  static const manHeight = 241.0;
+  static const manWidth = 236.0;
```

Comment:
don't think, we need width here. And pls think about making this dude size relative to the screen size.

Reply:
Yes, I'm deleted width  from here